Menu.py

```
@@ -0,0 +1,172 @@
+                    instructions()
+                elif quit_rect.collidepoint(event.pos):
+                    pygame.quit()
+                    sys.exit()
+
+def level_selection():
+    while True:
+        screen.fill(WHITE)
+        pygame.draw.rect(screen, BLACK, (0, 0, SCREEN_WIDTH, SCREEN_HEIGHT), 5)
+        draw_text("Select Level", font, BLACK, SCREEN_WIDTH // 2, 80)
+        
+        mouse_pos = pygame.mouse.get_pos()
```

Ruff (0.8.2)

Local variable `mouse_pos` is assigned to but never used

Remove assignment to unused variable `mouse_pos`

(F841)